### PR TITLE
Fix No such file or directory when attaching disk xml

### PR DESCRIPTION
**Pull request by chunfuwen**
By copying one disk xml for additional guest VM.
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

File: libvirt/tests/src/virtual_disks/virtual_disks_ceph.py [hunk 1]
```diff
@@ -761,7 +767,7 @@ def get_secret_list():
                 # Make sure the additional VM is running
                 if additional_vm.is_dead():
                     additional_vm.start()
-                ret = virsh.attach_device(guest_name, xml_file,
+                ret = virsh.attach_device(guest_name, additional_xml_file,
```
Comment (balamuruhans) [inline libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: The xml configuration is not changing for the attaching device for additional vm, then why would we copy xml to different name and attach ? I think existing code seems to be correct.

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: I noticed that original xml was deleted in real testing environment after it was once used as attachment parameter,but not quite why it happen, so just copy another xml for reattach.

Comment (balamuruhans) [reply libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: IMHO can you help debugging more on why it got deleted, ideally it should work. If there is some inconsistency in framework then we have to address the root cause rather working around in test case

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_ceph.py]: True, it is occasional issue,and take time. I will track it by creating one jira

**Pull request merged**